druid/src/ext_event.rs

```diff
@@ -90,8 +90,7 @@ impl ExtEventSink {
     /// instead you have to pass the [`Selector`] and the (optional) argument
     /// separately, and it will be turned into a `Command` when it is received.
     ///
-    /// The `obj` argument can be any type which implements `Any + Send`, or `None`
-    /// if this command has no argument.
+    /// The `object` argument must implement `Any + Send + Sync`.
```

Comment:
`object` argument -> `payload`
Also similar change for surrounding docs/variables.